release/release.go

```diff
@@ -139,6 +155,8 @@ func init() {
 
 	OnClassic = (ReleaseInfo.ID != "ubuntu-core")
 
+	HasSystemSeed = isCore20OrNewer()
```

Comment:
Unfortunately the real world will be more complicated, we have not restricted people using the core20 base to use the new layout only (right @pedronis ?). But maybe we don't need this (yet?) see  my comment above about the devicestate handling.

Reply:
yes, this code in release is premature and not what we are going to do,  the decision will be model assertion based.

Reply:
Removing and keeping it unset for now.